app/AppManifest.json

```diff
@@ -4,6 +4,9 @@
         "Port": 50008,
         "DAPR_GRPC_PORT": 50001,
         "Dockerfile": "./app/Dockerfile",
+        "vehicle_model": {
```

Comment:
@emsameen what do you say about this? Do you want snake case or a single word? What about `src`: I think it might be clearer than `path`

Reply:
Hi, yes. As discussed. "src" is OK and also changed that in the concept.

Reply:
camelCase would be:
vehicleModel

Reply:
vehiclemodel I don't see as good solution, I would apply VehicleModel

Reply:
As discussed, let's use **VehicleModel** for consistency and we need to verify all naming conventions for the app manifest concept.